python/lsst/meas/base/applyApCorr.py

```diff
@@ -207,8 +207,8 @@ def run(self, catalog, apCorrMap):
                 source.set(apCorrInfo.apCorrFlagKey, True)
                 oldFluxFlagState = False
                 if self.config.doFlagApCorrFailures:
-                    oldFluxFlagState = source.get(apCorrInfo.fluxFlagKey)
-                    source.set(apCorrInfo.fluxFlagKey, True)
+                    oldFluxFlagState = source.get(apCorrInfo.instFluxFlagKey)
+                    source.set(apCorrInfo.instFluxFlagKey, True)
```

Comment:
I think you and Jim B wanted a single flag to be shared by both kinds of flux, so shouldn't this remain "fluxFlagKey"?